service/history/api/updateworkflowoptions/api_test.go

```diff
@@ -45,13 +44,13 @@ var (
 	pinnedOverrideOptionsA = &workflowpb.WorkflowExecutionOptions{
 		VersioningOverride: &workflowpb.VersioningOverride{
 			Behavior:      enumspb.VERSIONING_BEHAVIOR_PINNED,
-			PinnedVersion: &deploymentpb.WorkerDeploymentVersion{DeploymentName: "X", BuildId: "A"},
+			PinnedVersion: "X/A",
```

Comment:
nit: if you call this slash `worker_versioning.WorkerDeploymentVersionIdDelimiter` then we won't have to change the test later

Reply:
Yeah, but there is something good about this: if someone changes the delimiter unexpectedly, this test will catch it. Will leave it as is until the delimiter discussion resolves.